src/actions.js

```diff
@@ -0,0 +1,35 @@
+// action types
+export const ActionTypes = {
+  UNDO: '@@redux-undo/UNDO',
+  REDO: '@@redux-undo/REDO',
+  JUMP_TO_FUTURE: '@@redux-undo/JUMP_TO_FUTURE',
+  JUMP_TO_PAST: '@@redux-undo/JUMP_TO_PAST',
+  JUMP: '@@redux-undo/JUMP',
+  CLEAR_HISTORY: '@@redux-undo/CLEAR_HISTORY'
+}
+// /action types
```

Comment:
I already dropped some of them in `helper.js`, but you're right, we can drop them here too.